main.css

```diff
@@ -94,15 +94,56 @@ footer
     background-color: #061428;
 }
 
-#sidebar
+#mobileMenuIcon
 {
     position: fixed;
+    top: 35px;
+    left: 5px;
+    width: 48px;
+    height: 48px;
+    z-index: 9999;
+    background: url("assets/mobile-ui/mobilemenu.png");
```

Comment:
You can assign me, but there's some discussion to be had about which icon font to use (FontAwesome actually has very limited relevant iconography, which I know because I started there when building the pin assets)